docs/application/native/guides/internationalization/i18n.md

```diff
@@ -773,6 +777,104 @@ The `current()` and `next()` functions only check the current index against the
+
+This discussion assumes that you use Plural Format with a predefined set of plural rules.
+You can create one using one of the constructors that takes a locale object.
+To specify the message pattern, you can either pass it to the constructor or set it explicitly
+using the `i18n_plural_format_apply_pattern()` function.
+The `i18n_plural_format_format_*` functions takes a number object and selects the message
+of the matching plural case. This message will be returned.
+
+### Patterns and Their Interpretation:
+
+The pattern text defines the message output for each plural case of the specified locale.  
+**Syntax:**
+
+```
+pluralStyle = [offsetValue] (selector '{' message '}')+
+offsetValue = "offset:" number
+selector = explicitValue | keyword
+explicitValue = '=' number  // adjacent, no white space in between
+keyword = [^[[:Pattern_Syntax:][:Pattern_White_Space:]]]+
+message: message = messageText (argument messageText)*
+argument = noneArg | simpleArg | complexArg
+complexArg = pluralArg | selectordinalArg
+noneArg = '{' argNameOrNumber '}'
+simpleArg = '{' argNameOrNumber ',' argType [',' argStyle] '}'
+pluralArg = '{' argNameOrNumber ',' "plural" ',' pluralStyle '}'
+selectordinalArg = '{' argNameOrNumber ',' "selectordinal" ',' pluralStyle '}'
+argNameOrNumber = argName | argNumber
+argName = [^[[:Pattern_Syntax:][:Pattern_White_Space:]]]+
+argNumber = '0' | ('1'..'9' ('0'..'9')*)
+argType = "number" | "date" | "time" | "spellout" | "ordinal" | "duration"
+argStyle = "short" | "medium" | "long" | "full" | "integer" | "currency" | "percent" | argStyleText
+```
+
+**Examples:**
+
+```
+one{dog} other{dogs}                        // English plural format for dog
+one{pes} two{psa} few{psi} other{psov}      //Slovenian translation of dog in plural format
+```
+
+Pattern_White_Space between syntax elements is ignored, except between the {curly braces}
+and their sub-message, and between the '=' and the number of an explicitValue.
+
+There are 6 predefined casekeyword in CLDR/ICU - 'zero', 'one', 'two', 'few', 'many' and 'other'.
+You always have to define a message text for the default plural case other
+which is contained in every rule set. If you do not specify a message text for a particular plural case,
+the message text of the plural case other gets assigned to this plural case.
+
+When formatting, the input number is first matched against the explicitValue clauses.
+If there is no exact-number match, then a keyword is selected by calling the Plural Rules
+with the input number minus the offset. (The offset defaults to 0 if it is omitted from the pattern string.)
+If there is no clause with that keyword, then the "other" clauses is returned.
+
+An unquoted pound sign (#) in the selected sub-message itself
+(i.e., outside of arguments nested in the sub-message) is replaced by the input number minus the offset.
+The number-minus-offset value is formatted using a Number Format for the Plural Format's locale.
+
+> **Note:**
```

Comment:
Please insert one empty line with `>` like below.

```
> **Note:**
>
> That argument is formatting without subtracting the offset!
```

To make a line break in notes, you should add two spaces **or** one empty line with `>`.
It is same to the end of line 869, that if you want to make a line break, you should fix it.

Reply:
Added two spaces at the end of 868 line